_posts/2021-03-23-connector-build.md

```diff
@@ -0,0 +1,232 @@
+---
+layout: post
+title:  "How to build your own KafkaConnect image via custom resource"
+date: 2021-03-23
+author: jakub_stejskal
+---
+In one of [the older blog post](https://strimzi.io/blog/2020/05/07/camel-kafka-connectors/) we shown you how easy is to integrate Camel with Strimzi via KafkaConnect custom resource.
```

Comment:
I would use _Camel Kafka Connectors_ and add a link. Maybe you can also format the KafkaConnect as `KafkaConnect`.